lib/rules/jsx-one-expression-per-line.js

```diff
@@ -59,8 +59,8 @@ module.exports = {
               return;
             }
 
-            countNewLinesBeforeContent = (child.raw.match(/^ *\n/g) || []).length;
-            countNewLinesAfterContent = (child.raw.match(/\n *$/g) || []).length;
+            countNewLinesBeforeContent = (child.raw.match(/^ *[\r\n|\r|\n]/g) || []).length;
+            countNewLinesAfterContent = (child.raw.match(/[\r\n|\r|\n] *$/g) || []).length;
```

Comment:
I think you meant `(\r\n|\r|\n)` or `[\r\n]` since brackets will match anything inside, even pipes. The best regex for line endings is simply `\r?\n`. It will match both CRLF and LF. Matching only CR is not needed as it's not used anymore.

Reply:
Thank you so much for the tip! Will try to fix it this weekend